### ISSUE TITLE
Request to be able to add custom resources contained in a jar file

### ISSUE THREAD

**Issue by jollyblade** [GradleUp/shadow#314]
### Shadow Version
latest 2.0.1

### Gradle Version
4.0.1

### Expected Behavior

This request is very specific to vaadin [https://vaadin.com/home].
Vaadin has a theme resource packaged in a vaadin dependency called vaadin-themes.
It has a VAADIN folder in the jarfile. The problem is, when during development a custom theme is created, it resides under: src/main/webapp/VAADIN/...

When building the fat jar for production, the resources from the vaadin-themes.jar/VAADIN should be packaged into the webapp/VAADIN folder. 

I tried relocate, but it did not help.
Is there a way to make it work in the current implementation?

Thank you very much!

**Comment by johnrengelman** [comment]
does this work when building a standard `jar` file? You'd have to configure Gradle to copy the `src/main/webapp` folder to the compile directory under the `webapp` folder.

**Comment by jollyblade** [comment]
No, basically this should be a war file, it's not very usual to deploy vaadin applications as fat jars, i guess.

**Comment by Goooler** [comment]
Outdated.
